submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/stats/domain/service/SubmissionStatsService.kt

```diff
@@ -0,0 +1,55 @@
+package ac.uk.ebi.biostd.stats.domain.service
+
+import ac.uk.ebi.biostd.persistence.common.model.SubmissionStatType
+import ac.uk.ebi.biostd.persistence.common.request.PaginationFilter
+import ac.uk.ebi.biostd.persistence.common.service.StatsDataService
+import ac.uk.ebi.biostd.stats.web.handlers.StatsFileHandler
+import ac.uk.ebi.biostd.stats.web.mapping.toStat
+import ac.uk.ebi.biostd.stats.web.mapping.toStatDto
+import ac.uk.ebi.biostd.stats.web.model.SubmissionStatDto
+import ac.uk.ebi.biostd.submission.domain.helpers.TempFileGenerator
+import org.springframework.web.multipart.MultipartFile
+
+class SubmissionStatsService(
```

Comment:
services need to return domain classes and we can convert to DTO in controller or add a mapper as DTO is a web cocnern.